scripts/get-versions.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+VERNUM='0|[1-9][0-9]*'
+
+# An example is "set-namespace/v1.2.3"
```

Comment:
Why do you need the name of the function instead of just `v.1.2.3`?

Reply:
The release is going to be git tag driven.
If we push a git tag `v1.2.3`, we can't tell which function this version is for.
The prefix in the git tag can be useful when there are multiple release targets in one repo.
This is similar to [what kustomize does](https://github.com/kubernetes-sigs/kustomize/tags).